src/components/atoms/Input/Input.model.ts

```diff
@@ -0,0 +1,7 @@
+export interface IProps {
+  type?: string;
```

Comment:
Probably to avoid typing anything here, we could use union and move only specific types here